client/src/components/LoginForm.js

```diff
@@ -7,7 +7,7 @@ import {
 } from 'reactstrap';
 import './LoginForm.css';
 
-const LoginForm = ({ onSubmit }: any) => {
+const LoginForm = ({ onSubmit }: Object) => {
```

Comment:
@rahil051 Object is not helpful either, it does not tell me what other properties this param can have

Reply:
@mashhoodr Object type is just for checking the props, which is destructing here.
There is a proposal for type checking for destructed variables
https://github.com/facebook/flow/issues/235

Reply:
@rahil051 the props should have a proper interface so we know exactly which properties are inside it, thats the whole point of type checking. So need to find out whats the best practice here.